yt_dlp/extractor/picarto.py

```diff
@@ -50,7 +51,7 @@ def _real_extract(self, url):
 
         if metadata.get('online') == 0:
             raise ExtractorError('Stream is offline', expected=True)
-        title = metadata['title']
+        title = metadata.get('title')
```

Comment:
Having extraction fail early if a `title` can't be extracted is an old youtube-dl convention that is no longer followed in yt-dlp, where `title` is not a hard requirement for video results anymore. When fixing ancient code like this, I prefer to inline `title` extraction in the `return` if possible.
```suggestion
```